pages/settings/upgrade-holiday-offer.tsx

```
@@ -0,0 +1,769 @@
+      <div className="mt-8 flex flex-col items-center space-y-2">
+        <a
+          target="_blank"
+          className="text-sm text-muted-foreground underline-offset-4"
+        >
+          All plans include unlimited viewers and page by page document
+          analytics.
+        </a>
+        <a
+          href="https://cal.com/marcseitz/papermark"
+          target="_blank"
+          className="text-sm text-muted-foreground underline-offset-4 hover:text-foreground hover:underline"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)